charts/ratify/values.yaml

```diff
@@ -14,6 +14,13 @@ cosignKeys: []
 
 notation:
   enabled: true
+  trustPolicies:
```

Comment:
please add all new values to readme , https://github.com/ratify-project/ratify/blob/dev/charts/ratify/README.md